### ISSUE TITLE
Google calendar integration

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/chapter#1044]
## What we currently do

At this time, all of Chapters interactions with users are via the app itself or email.

## Why this is an issue

It creates additional friction for users, since they will have to update their calendars manually. Potentially resulting in missed events and frustration.

## What we can do

Make use of the Google calendar api to update user's calendars for them.

## Potential issues

We'd need to investigate the api, integrate with Google auth, fallback to email for users that don't want to use Google's services and make sure that we store refresh tokens securely.

In addition, it's possible that we may need to verify with Google, depending on precisely which scopes we need.  Calendar is *probably* fine, but if we need to send emails through Google, that might require verification.

So, in conclusion this is potentially a little involved, but it would be great to get it in the MVP.  I recommend we tackle this last, though, simply because it will take a while to get right and we can beta test the email-only version while we do that.

**Comment by allella** [comment]
Here's the earlier conversation and research on updating Google Calendar
https://github.com/freeCodeCamp/chapter/issues/436

**Comment by ojeytonwilliams** [comment]
Closing in favour of the existing issue
